src/sync/sync_permission.cpp

```diff
@@ -261,23 +325,23 @@ SharedRealm Permissions::management_realm(std::shared_ptr<SyncUser> user, const
             Property{"id",                PropertyType::String, Property::IsPrimary{true}},
             Property{"createdAt",         PropertyType::Date},
             Property{"updatedAt",         PropertyType::Date},
-            Property{"expiresAt",         PropertyType::Date},
             Property{"statusCode",        PropertyType::Int|PropertyType::Nullable},
             Property{"statusMessage",     PropertyType::String|PropertyType::Nullable},
             Property{"token",             PropertyType::String|PropertyType::Nullable},
             Property{"realmUrl",          PropertyType::String},
             Property{"mayRead",           PropertyType::Bool},
             Property{"mayWrite",          PropertyType::Bool},
             Property{"mayManage",         PropertyType::Bool},
+            Property{"expiresAt",         PropertyType::Date|PropertyType::Nullable},
```

Comment:
Why is this moved to the end?

Reply:
I was trying to troubleshoot something; forgot to move it back. Thank you for catching this.